MVPMarsOnboarding/MVPMarsOnboarding/StepDefinitions/EducationModuleStepDefinition.cs

```diff
@@ -0,0 +1,113 @@
+﻿using MVPMarsOnboarding.Drivers;
+using MVPMarsOnboarding.Pages;
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using TechTalk.SpecFlow;
+
+namespace MVPMarsOnboarding.StepDefinitions
+{
+    [Binding]
+    public class EducationModuleStepDefinition:CommonDriver
+    {
+        ProfilePage profilePageObj;
+        LoginPage loginPageObj;
+        EducationModule educationModuleObj;
+
+        public EducationModuleStepDefinition()
+        {
+            driver = new ChromeDriver();
```

Comment:
initialise once in the Common driver

Reply:
just not you dont need this driver = new ChromeDriver(); as you have done this in common driver :)